content/concepts/false_positives_tuning.md

```diff
@@ -9,9 +9,25 @@ chapter: false
 
 ## What are False Positives?
 
-CRS provides _generic_ attack detection capabilities. A fresh CRS deployment has no awareness of the web services that may be running behind it, or the quirks of how those services work. It is possible that *genuine* transactions may cause some CRS rules to match in error, if the transactions happen to match one of the generic attack behaviors or patterns that are being detected. Such a match is referred to as a *false positive*, or false alarm.
+CRS provides _generic_ attack detection capabilities. A fresh CRS deployment has no awareness of the web services that may be running behind it, or the quirks of how those services work. Let's describe the possible scenarios with using an image:
+
+```mermaid
+%%{init: {"themeVariables": {"quadrant1Fill": "#00ff00", "quadrant2Fill": "#ff0000", "quadrant4Fill": "#ffff00"} }}%%
+quadrantChart
+    title WAF detection quadrant
+    x-axis Allowed --> Blocked
+    y-axis Normal Traffic --> Real attack
+    quadrant-1 True Positive
+    quadrant-2 False Negative 
+    quadrant-3 Business as usual
+    quadrant-4 False Positive
+```
+
+Let's begin with describing a perfect world. In theory, CRS should _allow all valid requests_. This can be seen above as `Business as usual`. Also, CRS should only block attacks. This is a `True Positive` match, and when this happens CRS is doing its job.
+
+Two additional scenarios exist. It is possible that *genuine* transactions may cause some CRS rules to match in error, if the transactions happen to match one of the generic attack behaviors or patterns that are being detected. Such a match is referred to as a `False Positive`, or false alarm. The worse case is when there is a real attack, and CRS is not able to detect it. This is known as a `False Negative`.
```

Comment:
```suggestion
In reality, no system is ideal and CRS will sometimes _block a valid request_ (`False Positive`) or _allow a real attack_ (`False Negative`). Obviously, false negatives are the worse of the two because of the potential repercussions of a successful intrusion.
```